keel-sql/src/main/resources/db/changelog/20200714-add-event-application-column.yml

```diff
@@ -0,0 +1,28 @@
+databaseChangeLog:
+- changeSet:
+    id: add-event-application-column
+    author: fletch
+    changes:
+    - sql:
+        sql: |
+          alter table event add column application varchar(255) generated always as (json ->> '$.application') stored;
+    - createIndex:
+        tableName: event
+        indexName: event_ref_scope_application_timestamp_idx
+        columns:
+        - column:
+            name: ref
+        - column:
+            name: scope
+        - column:
+            name: application
+        - column:
+            name: timestamp
+            descending: true
+    - dropIndex:
+        tableName: event
+        indexName: event_uid_scope_timestamp_idx
+    rollback:
```

Comment:
The rollback currently doesn't handle the added and dropped indexes.

Reply:
You get auto-rollback when you use Liquibase's defined migration types